powershell/internal/Test-MtAzContext.ps1

```diff
@@ -0,0 +1,22 @@
+﻿<#
+.SYNOPSIS
+    Validates the AzContext to ensure a valid connection to Azure.
+    This cmdlet is used mainly by the Get-MtMaesterApp and Update-MtMaesterApp cmdlets and related cmdlets.
+#>
+
+function Test-MtAzContext {
+    [CmdletBinding()]
+    [OutputType([bool])]
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingWriteHost', '', Justification = 'Colors are beautiful')]
+    param ()
+
+    $validContext = $true
```

Comment:
Should this be swapped to start without a valid context and only set it to $true if the context is confirmed?
Are any permissions required?